src/main/java/lotto/controller/LottoApplicationController.java

```diff
@@ -4,61 +4,42 @@
 import java.util.List;
 
 import lotto.domain.Lotto;
-import lotto.domain.LottoNumberFactory;
+import lotto.domain.LottoNumber;
 import lotto.domain.LottoNumbers;
-import lotto.domain.LottoStatistics;
-import lotto.domain.PurchaseAmount;
-import lotto.domain.WinningNumbers;
+import lotto.domain.LottoShop;
+import lotto.domain.Lottos;
 
 public class LottoApplicationController {
-	private static final int PURCHASE_FINISH = 0;
-	private PurchaseAmount purchaseAmount;
-	private WinningNumbers winningNumbers;
-	private LottoStatistics lottoStatistics = new LottoStatistics();
-	private List<Lotto> lottos = new ArrayList<>();
 
-	public String validatePurchaseAmount(String purchaseAmount) {
-		try {
-			this.purchaseAmount = new PurchaseAmount(purchaseAmount);
-		} catch (IllegalArgumentException exception) {
-			return exception.getMessage();
-		}
-		return "";
-	}
+	public static final int LOTTO_PRICE = 1000;
+	private static final int PURCHASE_FINISH = 0;
+	private Lottos lottos;
 
-	public void purchaseLotto() {
-		int purchaseQuantity = getPurchaseQuantity();
+	public void purchaseLotto(int purchaseQuantity) {
+		List<Lotto> lottos = new ArrayList<>();
 		while (continuePurchase(purchaseQuantity)) {
-			lottos.add(new Lotto(new LottoNumbers(LottoNumberFactory.create())));
+			lottos.add(LottoShop.sell());
 			purchaseQuantity--;
 		}
+		this.lottos = new Lottos(lottos);
 	}
 
 	private boolean continuePurchase(int purchaseQuantity) {
 		return purchaseQuantity > PURCHASE_FINISH;
 	}
 
-	public String validateWinningNumbers(String winningNumbers) {
-		try {
-			this.winningNumbers = new WinningNumbers(winningNumbers);
-		} catch (IllegalArgumentException exception) {
-			return exception.getMessage();
-		}
-		return "";
-	}
-
-	public LottoStatistics recorde() {
-		for (Lotto lotto : lottos) {
-			lottoStatistics.record(lotto.countMatchNumber(winningNumbers));
+	public List<Lotto> recorde(String winningNumbers, int bonusBallNumber) {
+		for (Lotto lotto : lottos.getLottos()) {
+			lotto.recordeRank(new LottoNumbers(winningNumbers), new LottoNumber(bonusBallNumber));
 		}
-		return lottoStatistics;
+		return lottos.getLottos();
 	}
 
-	public int getPurchaseQuantity() {
-		return purchaseAmount.purchase();
+	public int getPurchaseQuantity(int purchaseAmount) {
+		return purchaseAmount / LOTTO_PRICE;
```

Comment:
원시 값을 포장하는 Money와 같은 클래스를 도출해보는 것은 어떨까요 ? 
Rank의 amount나 상수로 관리하던 LOTTO_PRICE와 같은 금액을 의미했던 int 타입들을 포장할 수 있을 것 같아요!
Money 클래스는 어떤 적합한 책임을 할당할 수 있을 지 고민해보시면 좋을 것 같아요 👍 


> 규칙 3: 모든 원시값과 문자열을 포장한다.

```suggestion
	public int getPurchaseQuantity(Money purchaseAmount) {
		return purchaseAmount.divide(LOTTO_PRICE);
```